src/ctia/flows/crud.clj

```diff
@@ -99,14 +99,16 @@
     (when spec
       (when-not (cs/valid? spec entity)
         (throw (ex-info (cs/explain-str spec entity)
```

Comment:
We should not throw an exception for `bulk` or `bundle` operations. Some entities could be created before the spec error occurs.